docker-compose.yml

```diff
@@ -25,8 +25,11 @@ services:
       PGDATA: /var/lib/postgresql/data/pgdata
     volumes:
       - dbdata:/var/lib/postgresql/data
-    networks:
+    ports:
+      - ${DB_PORT:-0}:5432
```

Comment:
will this create a mapping on each container? Or just on the one where a DB_PORT is specified?

I'll run many containers, which is why I'm running without port mapping to the host network.

Reply:
If DB_PORT is set, it will fix it for all. If not, Docker will assign a random port.
The link is reported on container startup on both cases